src/system/Footer/Footer.test.tsx

```diff
@@ -0,0 +1,48 @@
+import '@testing-library/jest-dom';
+import { render, screen } from '@testing-library/react';
+
+/**
+ * Internal dependencies
+ */
+import { Footer } from './Footer';
+
+const links = [
+	{
+		children: 'Link1',
+		href: 'https://wpvip.com/',
+	},
+	{
+		children: 'Link2',
+		href: 'https://docs.wpvip.com/',
+	},
+];
+
+describe( '<LinkExternal />', () => {
+	it( 'should render LinkExternal props', () => {
```

Comment:
should render Footer?

Reply:
I'll update it to be more clear, maybe to `should accept LinkExternal props for Footer links` as I wanted to make sure that props for `LinkExternal` can be passed without error to the `Footer`